### DISCUSSION TITLE
Clue Scroll Plugin: Allow note editing without needing the step in your inventory

### DISCUSSION THREAD

**Discussion by acdyer** [runelite/runelite#17665] [Ideas]
I've been using the notes feature in the clue scroll overlay for over a year now. I write in the teleport I use to reach the step. It's taken a very long time to fill them all in, because I can't fill each in until I get that specific clue step.

With the Varlamore release, all of my Jewellery box notes are now wrong, since the new dueling ring teleport shifted everything by one. For example, a clue step I had previously marked as "Jewellery box C" now needs to read "Jewellery box D". I can't find any way to go and fix all of them at once, though. I'm only able to edit a clue step when I have it in my inventory. So it will take months or years before I have them all fixed.

It would be so helpful if there was a way to access these notes all in one place, so I can just fix them all at once.

**Comment by Nightfirecat** [comment]
We don't have a good way to implement that on a technical level. I might recommend, however [using the Better Teleport Menu hub plugin](https://runelite.net/plugin-hub/show/better-teleport-menu) to reassign those binds to their original values so you won't need to update your clue notes.
